demo/src/pages/index.js

```diff
@@ -84,6 +43,26 @@ const IndexPage = ({ data }) => {
       <GlobalHeader
         editUrl={`${siteMetadata.repository}/tree/develop/demo/src/pages/index.js`}
       />
+<header
```

Comment:
It looks like prettier got out of whack here. Could you run formatting again on this file?